build.gradle

```diff
@@ -69,6 +69,9 @@ sourceSets {
 repositories {
     // You can declare any Maven/Ivy/file repository here.
     mavenCentral()
+    maven {
+        url "https://plugins.gradle.org/m2/" // Added for SonarQube
```

Comment:
This should only bee needed for the legacy configuration, see [this documentation page](https://plugins.gradle.org/plugin/org.sonarqube). Plus it's in the wrong section. I believe we should remove it.

Reply:
Yep that makes sense, for whatever reason I was getting a error saying and added it. However, not it seems that error is gone. I've removed it now.